etc/profile-m-z/tutanota-desktop.profile

```diff
@@ -9,6 +9,9 @@ include globals.local
 noblacklist ${HOME}/.config/tuta_integration
 noblacklist ${HOME}/.config/tutanota-desktop
 
+# breaks tray functionality and secret storage
```

Comment:
Is it possible to use dbus filtering instead of opening it all up? You can use d-feet for example to check what tutanota exposes.

Reply:
> ```diff
> +# breaks tray functionality and secret storage
> +ignore dbus-user none
> ```

See `allow-tray` in firejail.config and `?ALLOW_TRAY` in profiles.

By "breaks secret storage" do you mean that it cannot access things like KDE
Wallet?

Reply:
So even with something like the below, that 'secret storage' is broken?
```
dbus-user filter
dbus-user.talk org.freedesktop.Notifications
dbus-user.talk org.freedesktop.secrets
dbus-user.talk org.gnome.keyring.SystemPrompter
#dbus-user.talk org.mozilla.Firefox.*
#dbus-user.talk org.mozilla.firefox.*
ignore dbus-user none
```

Reply:
I think that xdg-open fails because it's a shell script and the `include disable-shell.inc` blocks that. You can try adding `include allow-bin-sh.inc` to get that working. This would obviously imply adding `sh` to private-bin.

One more remark. I'm trying to register a free account to further test the tutanota-desktop app on my side. I've installed it on my arch linux box and it starts okay, but due to https://github.com/tutao/tutanota/issues/4342 I haven't gotten a working login yet. Contacted their support about this and will update my comment as soon as I get a reply from them.